aten/src/ATen/native/MaxUnpooling.cpp

```diff
@@ -77,20 +77,19 @@ static void max_unpooling3d_shape_check(
       "elements in indices should be type int64");
   TORCH_CHECK(
       (input.ndimension() == 4 || input.ndimension() == 5),
-      "Input to max_unpooling3d should be a 4d or 5d Tensor",
-      input.sizes());
+      "Input to max_unpooling3d should be a 4d or 5d Tensor, but got a tensor with dim ", input.ndimension());
```

Comment:
nit dim -> X dimensions

Reply:
My bad, thanks Alban for pointing this out. Should be fixed now.